### ISSUE TITLE
t_blueprint_mesh_verify fails on Windows

### ISSUE THREAD

**Issue by agcapps** [LLNL/conduit#635]
After @bmhan12 's fixes in #632 , I recompiled using MS Visual Studio and still had failures:

- Debug: test topology_unstructured reports failure.
  - Any call to CHECK_MESH()
    - after setting `n["subelements"][ (something) ]`, expecting the verify function to return `false`
    - or at lines 734 or 738 (also expect the verify function to return `false`)
- Release: all tests report success.

**Comment by agcapps** [comment]
@cyrush , @xjrc , @bmhan12 , these are the remaining failures I see on Windows after #623 is resolved.

**Comment by cyrush** [comment]
@agcapps I think this is now resolved, I just merged #636 and tested a debug build on windows and the tests passed. 

If there is anything else we should try to fix, can you throw up a new issue?

**Comment by agcapps** [comment]
> 
> 
> @agcapps I think this is now resolved, I just merged #636 and tested a debug build on windows and the tests passed.
> 
> If there is anything else we should try to fix, can you throw up a new issue?

Thanks!  Looks great.  I'll put in a new issue if anything else crops up (which it hasn't so far).
